.github/workflows/crowdin-sync.yml

```diff
@@ -0,0 +1,29 @@
+name: Crowdin Synchronisation
```

Comment:
I wonder if we should create a plugin for this workflow. We've done it for PR linter to avoid copy pasting the same code everywhere, and it is also easier to maintain in one place. We are using crowdin variables here though, maybe it is an argument against the plugin approach 🤷‍♀️ But not sure.